toolkit/scripts/setuplkgtoolchain.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+ROOT_FOLDER=$(git rev-parse --show-toplevel)
+
+usage() {
+    echo "./setuplkgtoolchain.sh"
+    echo " Syncs toolchain manifests to match LKG build"
+    exit 1
+}
+
+getlkg() {
+    wget -O lkg-3.0-dev.json -nv https://mariner3dailydevrepo.blob.core.windows.net/lkg/lkg-3.0-dev.json
+    DAILY_BUILD_ID=$(cat lkg-3.0-dev.json | jq -r .dailybuildid | tr . -)
+    GIT_COMMIT=$(cat lkg-3.0-dev.json | jq -r .commit)
+}
+
+checkformodifiedmanifests() {
+    if [[ $(git status --porcelain | grep pkggen_core_$(uname -p).txt | wc -l) -ne "0" ||
+          $(git status --porcelain | grep toolchain_$(uname -p).txt | wc -l) -ne "0" ]]; then
```

Comment:
Idea to shorten the condition:
```suggestion
    if git status --porcelain | grep -qP "(pkggen_core|toolchain)_$(uname -p).txt"; then
```

Reply:
thanks!